testData/actions/generation/generator/ModuleXmlGenerator/generateModuleFile/module.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Test_Module"/>
+    <module name="Test_Module" setup_version="1.0.0"/>
```

Comment:
Please leave these tests as they are, just add a new version config. For the case with `setup_version` we need a new test. Both cases should be covered though. Thanks!